src/backend/api/reviews.js

```diff
@@ -0,0 +1,96 @@
+const { query } = require("express");
+const express = require("express");
+const { max } = require("../database");
+const router = express.Router();
+const knex = require("../database");
+
+router.get('/', async (req, res) => {
+    try {
+        const reviewList = await knex('review').select('*')
+        if (!reviewList.length) {
+            res.status(404).json("There are not available any reviews")
+        } else {
+            res.json(reviewList);
+        }
+    } catch (error) {
+        throw res.status(404).json({ error: "There is error in Page try again" });
+    }
+});
+
+router.get('/:id', async (req, res) => {
+    try {
+        const id = req.params.id
+        const reviewList = await knex.from('review').select('*').where('meal_id', id);
+        if (!reviewList.length) {
+            res.status(404).json("There are not available any reviews")
+        } else {
+            res.json(reviewList);
+        }
+    } catch (error) {
+        throw res.status(404).json({ error: "There is error in Page try again" });
+    }
+})
+router.post('/', async (req, res) => {
+    try {
+        const insertData = await knex('review').insert({
+            title: req.body.title,
+            description: req.body.description,
+            meal_id: req.body.meal_id,
+            stars: req.body.stars,
+            created_date: req.body.created_date,
```

Comment:
I would remove this line because the database should set it by default.

Reply:
I don't get in this comment. How can make post req.  without passing value

Reply:
Because the database has a default value for this column, you don't have to set it here. If you remove this line, you should see that `created_date` will still be set with the default value.